### PR TITLE
(Feature) Automate dkim records

### PR DESCRIPTION
**Pull request by Stretch96**

### CODE REVIEW

**Comment by simonrondelez** [comment]
hi @Stretch96 thank you for this contribution. Could you give a small description in the PR on what your goal is and also update the documentation accordingly so it matches the code?

File: main.tf [hunk 1]
```diff
@@ -1,3 +1,14 @@
+provider "aws" {
```
Comment (iuriaranda) [inline main.tf]: Why do you need an extra provider for the dkim resources?

Comment (Stretch96) [reply main.tf]: SES can only be deployed in a few regions, so it's highly likely that this will need to be changed when using this module

Comment (iuriaranda) [reply main.tf]: true, but that'll be the case for all the resources in the module, not just `aws_ses_domain_identity` and `aws_ses_domain_dkim`, no?

Also, you can override the provider for the whole module externally: https://www.terraform.io/docs/modules/usage.html#passing-providers-explicitly

Comment (Stretch96) [reply main.tf]: Ah yep, I see what I've done here now 😑 

I was trying to ensure only the SES resources would launch in the few regions available, but still allow the other resources to launch in the default region. However, Route53 records are Global, so it doesn't matter in this case 👍 ... And right now I can't think of other resources that'd be needed for this module that the user would want to keep in the default region

(eg. If Route53 resources were region specific, you wouldn't be able to use a zone id not in the region specified for SES)

I'll modify the PR 👍

Comment (iuriaranda) [reply main.tf]: Ok, thanks for the clarification, I now understand why you added that 🙂 
And thanks for taking the time to modify the PR

**Comment by Stretch96** [comment]
@iuriaranda @simonrondelez I've modified the PR now and added some info to the Readme 👍

**Review by iuriaranda — APPROVED** [review]
Looks good. 🙂  Thanks for your contribution!

We require two approvals to merge a PR, so I'll request someone from the team to review it as well.

**Review by venturel — APPROVED** [review]
Looks good now. Thanks

**Review by SamClinckspoor — APPROVED** [review]

**Pull request merged**